devel/vscode_settings.md

```diff
@@ -0,0 +1,21 @@
+## Automatic organizing of typescript imports
+
+Use the following in `~/.config/Code/User/settings.json` to achieve automatic organizing of typescript imports:
+
+```
+{
+    ...
+    "[typescript]": {
+        "editor.codeActionsOnSave": {
+            "source.organizeImports": true
+        }
+    },
+    "[typescriptreact]": {
+        "editor.codeActionsOnSave": {
+            "source.organizeImports": true
+        }
+    }
+}
+```
```

Comment:
Didn't you show me a way to do this through the UI rather than editing the config file?

Reply:
yes, but when I reinstalled my OS and everything I had trouble finding my way back. But this is the right way to do it because it doesn't require an extension - built into vscode